vite.config.ts

```diff
@@ -1,13 +1,19 @@
 import { defineConfig } from 'vite'
+import eslintPlugin from 'vite-plugin-eslint'
+
 import Icons from 'unplugin-icons/vite'
 import { FileSystemIconLoader } from 'unplugin-icons/loaders'
 import IconResolver from 'unplugin-icons/resolver'
 import Components from 'unplugin-vue-components/vite'
-import eslintPlugin from 'vite-plugin-eslint'
+import vue from '@vitejs/plugin-vue'
+
+import type { UserConfigExport } from 'vitest/config'
+import { nuxtAliases } from './nuxt.config'
 
 // Used in Vitest
-export const ViteConfig = {
+export const ViteConfig: UserConfigExport = {
   plugins: [
+    vue(),
```

Comment:
Noticed the relevant removed line in `vitest.config.ts`; do we not need the `vueJsx` plugin anymore?